ascii_transport_format.py

```diff
@@ -1,18 +1,19 @@
 import json
 
 class ASCIITransportFormat:
-
     SUPPORTED_TYPES = ['FILE', 'JSON', 'STRING']
 
     def __init__(self, type=None, data=None, encoded:bool=False) -> None:
```

Comment:
Use a different name for `type` (suggestion: `format`); `type` is a keyword in python

Reply:
🐳